src/help.cpp

```diff
@@ -251,6 +251,14 @@ void add_options() {
   opt->help = "Number of initial records in table";
   opt->setInt(1000);
 
+  /* Initial Records in table */
+  opt = newOption(Option::BOOL, Option::EXACT_INITIAL_RECORDS,
+                  "exact-initial-records");
+  opt->help = "Insert exact number of initial records mention with --records. "
+              "Otherwise it randomly insert records in each table";
+  opt->setBool(false);
+  opt->setArgs(no_argument);
```

Comment:
The new option --exact-initial-records must not accept any argument. It does at the moment. Please check.

Reply:
./pstress-ms  --socket $SOCKET --table 4 --records 100 --no-table-compression --exact-initial-records=1 --seconds 100 --log-all-queries
pstress-ms: option `--exact-initial-records' doesn't allow an argument
Invalid option , exiting
{Rahuls-MacBook-Pro x28 o8 src}$

Reply:
./pstress-ps --tables 1 --logdir=`pwd`/log --threads 10 --socket /tmp/mysql_22000.sock --seconds 0 --records 200 --exact-initial-records 1000
- Connecting to default.node.tld []...
metadata created randomly
The value of rec:200
The value of rec:200
starting random load in 10 threads.
Saving metadata to file /home/mohit.joshi/pstress_8.0/src/log/step_1.dll
COMPLTED

Run without "="

Reply:
all other binary options are also like that. for e.g check --jlddl. the behaviour is fine.